x-pack/plugin/eql/qa/common/src/main/resources/test_between_queries.toml

```diff
@@ -0,0 +1,11 @@
+[[queries]]
```

Comment:
why not  just add these to `test_queries_supported.toml`?

Reply:
Sure can rename the file and we could start using it for all the extra queries that are not included in the original test_queries.toml. Will do that.